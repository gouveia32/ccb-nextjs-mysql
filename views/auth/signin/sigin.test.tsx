import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import SignInPage, { SignInProps } from "../../../pages/auth/signin";

describe("SignIn page", () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    const mockProps: SignInProps = {
      providers: null,
    };

    wrapper = shallow(<SignInPage {...mockProps} />);
  });

  it("should render SignIn page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
