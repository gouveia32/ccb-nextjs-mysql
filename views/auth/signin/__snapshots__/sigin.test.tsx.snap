// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn page should render SignIn page 1`] = `ShallowWrapper {}`;
